src/core/lib/security/security_connector/ssl/ssl_security_connector.cc

```diff
@@ -262,15 +262,23 @@ class grpc_ssl_server_security_connector
       size_t num_alpn_protocols = 0;
       const char** alpn_protocol_strings =
           grpc_fill_alpn_protocol_strings(&num_alpn_protocols);
-      const tsi_result result = tsi_create_ssl_server_handshaker_factory_ex(
-          server_credentials->config().pem_key_cert_pairs,
-          server_credentials->config().num_key_cert_pairs,
-          server_credentials->config().pem_root_certs,
+      tsi_ssl_server_handshaker_options options;
+      memset(&options, 0, sizeof(options));
```

Comment:
Can you please add proper initial values to `tsi_ssl_server_handshaker_option` so that there is no need to memset it?